pkg/proxy/config/config.go

```diff
@@ -62,6 +64,22 @@ type EndpointsHandler interface {
 	OnEndpointsSynced()
 }
 
+// EndpointSliceHandler is an abstract interface of objects which receive
+// notifications about endpoint slice object changes.
+type EndpointSliceHandler interface {
```

Comment:
I didn't have one because there is no difference between add and update at this point, but I've added one now for more consistency and ease of expansion in the future.